README.md

```diff
@@ -336,6 +338,13 @@ Unsubscribe from a topic or topics
 * `callback` - `function (err)`, fired on unsuback. An error occurs if client is disconnecting.
 
 -------------------------------------------------------
+<a name="startConnection"></a>
+### mqtt.Client#startConnection()
+
+Manually starts the connection.
+Use it when `delayConnection` has been passed to the constructor.
+
+-------------------------------------------------------
```

Comment:
how about we call it `start`? and maybe call the option `start`? I think it might play better with the rest of the API, specifically about `end()`.